app/assets/javascripts/admin/onboarding.js

```diff
@@ -121,12 +135,21 @@ export class InviteUsersPopover extends React.Component<{
 }
 
 const OrganizationForm = Form.create()(({ form, onComplete }) => {
+  const [organizations, setOrganizations] = useState([]);
+  const fetchOrganizations = async () =>
+    setOrganizations((await getOrganizations()).map(org => org.name));
+  useEffect(() => {
```

Comment:
`useEffect(fetchOrganizations, [])` should work, too, right?

Reply:
Unfortunately not ^.~

> There is one last catch. In the code, we are using async/await to fetch data from a third-party API. According to the documentation every function annotated with async returns an implicit promise: “The async function declaration defines an asynchronous function, which returns an AsyncFunction object. An asynchronous function is a function which operates asynchronously via the event loop, using an implicit Promise to return its result. “. However, an effect hook should return nothing or a clean up function. That’s why you may see the following warning in your developer console log: 07:41:22.910 index.js:1452 Warning: useEffect function must return a cleanup function or nothing. Promises and useEffect(async () => …) are not supported, but you can call an async function inside an effect.. That’s why using async directly in the useEffect function isn’t allowed. Let’s implement a workaround for it, by using the async function inside the effect.

from https://www.robinwieruch.de/react-hooks-fetch-data/

Reply:
Argh, ok, this will still bite me a few times probably.